rust/gui-client/src-tauri/src/client/logging.rs

```diff
@@ -97,21 +107,22 @@ pub(crate) async fn count_logs() -> StdResult<FileCount, String> {
 ///
 /// This includes the current log file, so we won't write any more logs to disk
 /// until the file rolls over or the app restarts.
-pub(crate) async fn clear_logs_inner(managed: &Managed) -> Result<()> {
-    let mut dir = tokio::fs::read_dir(log_path()?).await?;
-    while let Some(entry) = dir.next_entry().await? {
-        tokio::fs::remove_file(entry.path()).await?;
+pub(crate) async fn clear_logs_inner() -> Result<()> {
+    for log_path in log_paths()?.into_iter().map(|x| x.src) {
+        let mut dir = tokio::fs::read_dir(log_path).await?;
+        while let Some(entry) = dir.next_entry().await? {
+            tokio::fs::remove_file(entry.path()).await?;
+        }
     }
 
-    managed.fault_msleep(5000).await;
     Ok(())
 }
 
 /// Pops up the "Save File" dialog
 pub(crate) fn export_logs_inner(ctlr_tx: CtlrTx) -> Result<()> {
     let now = chrono::Local::now();
     let datetime_string = now.format("%Y_%m_%d-%H-%M");
-    let stem = PathBuf::from(format!("connlib-{datetime_string}"));
+    let stem = PathBuf::from(format!("firezone_logs_{datetime_string}"));
```

Comment:
Changed the default zip name to match the macOS Client.

I could also make it default to the Documents folder if that's worth it for UX.